src/FileStorageBase/Cache/ClientCache/MultiClient.cs

```diff
@@ -54,7 +54,7 @@ private Client Client(Network.Address address)
 
         private void IterateClients(Action<Client> handler, CancellationToken token)
         {
-            string errMsg = "";
+            Exception lastErr = null;
             foreach (var address in addresses)
```

Comment:
With only one call it's enough? we don't need to call handler for each address?

Reply:
All this addresses are belong to one node. Only one success is enough.